pom.xml

```diff
@@ -59,6 +59,11 @@
 			<version>5.1.0</version>  <!-- ✅ Latest stable as of 2025 -->
 		</dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
```

Comment:
Adding ```spring-boot-starter-security``` will secure all endpoints. We don't want it right now. Instead, use ```spring-security-core```. This will allow you to use BCrypt alogrithm and still keep endpoints open.